validate/src/message.rs

```diff
@@ -203,8 +203,18 @@ pub fn embeds(embeds: &[Embed]) -> Result<(), MessageValidationError> {
             source: None,
         })
     } else {
+        let mut total = 0;
         for (idx, embed) in embeds.iter().enumerate() {
-            crate::embed::embed(embed).map_err(|source| {
+            total += crate::embed::_embed(embed).map_err(|source| {
```

Comment:
this repetition could be avoided with `impl From<(EmbedValidationError, usize)> for MessageValidationError` but i haven't seen this pattern somewhere else in the crate

Reply:
It's not for errors, but `ClusterBuilder` does impl `From<(String, Intents)>`, so it is a pattern we use

Reply:
should i commit that then? doing that just to avoid a map_err() repetition seems a bit extreme to me